addons/mail/static/src/models/file_uploader_view/file_uploader_view.js

```diff
@@ -45,6 +182,7 @@ registerModel({
             isCausal: true,
             readonly: true,
         }),
+        fileInputRef: attr(),
```

Comment:
Maybe it is out of scope of the current PR, but ideally we would like to get rid of the component because it doesn't display anything.

Instead if possible we should try to put the `<input>` into a fragment created from the model directly and do everything from it.

Reply:
I understand what you wish, but that it could be done in an other PR